### DISCUSSION TITLE
Cannot limit the current of SerialBattery

### DISCUSSION THREAD

**Discussion by feldsoft** [Louisvdw/dbus-serialbattery#1065] [Q&A]
Hello,
cannot limit the current of the SerialBattery. I have set MAX_BATTERY_CHARGE_CURRENT = 5.0. This is also displayed under the SerialBattery parameter. Aggregated in BatteryAggregator  parameter. But it puts what the panels have into the batteries. I put the Cerbo to the factory settings and reinstalled it. Same behavior. I took out the aggregator and a battery and tried it with just one battery. Again, no current limit.

Am I misunderstanding something or have I overlooked a setting?

I'm thankful for every hint.

Here are some info of my system
[Current.pdf](https://github.com/Louisvdw/dbus-serialbattery/files/15414381/Current.pdf)

My system:
3 x MPII 5000
3 x charge controller MPPT 250/70, 2 x MPPT 150/60 with a total of 7,400 Wp
2 x Hoymiles HM 600 integrated into the Cerbo 1480 Wp via OpenDTU
2 x 16S batteries with EVE 280Ah cells, JBD BMS connected to the Cerbo via RS 485 / USB, the data then processed via dbus-serialBattery and aggregated with the BatteryAggregator.
1 x EM24
1 x Victron 500 A shunt
Cerbo v3.31
SerialBattery 1.2.20240401
BatteryAggregator 3.0.61

Alfred

**Comment by MatKra89** [comment]
The excess feed-in may have been selected in the Victron system, then the current limitation does not work. You can deactivate this as a test.
Greetings

**Comment by feldsoft** [comment]
Thank you very much for the answer. The switch is off, as you see in the PDF.

My problem was, that the cell diff was to high. Now I have made something in Node Red. Everything is fine in this case.
But I don't understand that serial battery don't work in my case. I'm sure, it is my problem, but I would like to understand that.

Thank you Alfred
